workitem_blackbox_test.go

```diff
@@ -326,6 +329,36 @@ func getWorkItemTestData(t *testing.T) []testSecureAPI {
 			payload:            nil,
 			jwtToken:           "",
 		},
+		// adding security tests for workitem.2 endpoint
+		{
+			method:             "PATCH",
```

Comment:
helpful, replaced all such instances from that file in https://github.com/almighty/almighty-core/pull/482/commits/1c74a7a920d535fcf5e5ef3254b85dbc88b06a7a